docs/pages/installation.mdx

```diff
@@ -4,7 +4,11 @@ description: The guide for installing Teleport on servers and into Kubernetes cl
 h1: Installation
 ---
 
-Teleport core service [`teleport`](./setup/reference/cli.mdx#teleport) and admin tool [`tctl`](./setup/reference/cli.mdx#tctl) have been designed to run on **Linux** and **Mac** operating systems. The Teleport user client [`tsh`](./setup/reference/cli.mdx#tsh) and UI are available for **Linux, Mac**, and **Windows** operating systems.
+The Teleport daemon [`teleport`](./setup/reference/cli.mdx#teleport) and admin
```

Comment:
With this placement, I first thought this was intended to be an introductory paragraph. To prevent other users from having the same assumption and glossing over this paragraph, what would you think of:

- Adding a heading (e.g., "## Supported operating systems") below the "FIRST TIME TRYING TELEPORT" Admonition)
- Moving this information into a table in the new section